### PR TITLE
feat: onboarding todo

### PR DESCRIPTION
**Pull request by reneaaron**

### CODE REVIEW

File: frontend/src/screens/wallet/OnboardingChecklist.tsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    hasChannelManagement &&
+    albyBalance &&
+    albyBalance.sats * (1 - ALBY_SERVICE_FEE) >
+      ALBY_MIN_BALANCE + 50000; /* accomodate for onchain fees */
+  const isLinked =
+    albyMe &&
+    nodeConnectionInfo &&
+    albyMe?.keysend_pubkey === nodeConnectionInfo?.pubkey;
+  const hasChannel = hasChannelManagement && channels && channels?.length > 0;
+  const hasBackedUp =
```
Comment (reneaaron) [inline frontend/src/screens/wallet/OnboardingChecklist.tsx:37]: Is there a way to know if a user already did a backup? I just found the backup reminders that seemed to come back after some days...

File: frontend/src/screens/wallet/OnboardingChecklist.tsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+      to: "/onboarding/lightning/migrate-alby",
+    },
+    {
+      title: "Connect your first app",
+      description:
+        "Seamlessly connect apps and integrate your wallet with other apps from your hub.",
+      checked: hasCustomApp,
+      to: "/appstore",
+    },
+    {
```
Comment (rolznz) [inline frontend/src/screens/wallet/OnboardingChecklist.tsx:72]: this should only be included if `hasMnemonic` is true

File: frontend/src/screens/wallet/OnboardingChecklist.tsx (hunk 1 above)
Comment (rolznz) [reply frontend/src/screens/wallet/OnboardingChecklist.tsx:37]: No, but maybe it's fine for now. It'll only show once every 3 months

File: frontend/src/screens/wallet/OnboardingChecklist.tsx [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+      to: "/channels/new",
+    },
+    {
+      title: "Link your Alby Account",
+      description: "Link your lightning address & other apps to this hub.",
+      checked: isLinked,
+      to: "/settings",
+    },
+    {
+      title: "Migrate your balance to your Alby Hub",
```
Comment (rolznz) [inline frontend/src/screens/wallet/OnboardingChecklist.tsx:60]: This one I'm unsure about. It gets marked even if you haven't migrated anything, or if you have less than the minimum amount to open a channel but more than 100 sats remaining

Comment (rolznz) [reply frontend/src/screens/wallet/OnboardingChecklist.tsx:60]: I wonder if we should remove this one for now to avoid confusion?

Comment (rolznz) [reply frontend/src/screens/wallet/OnboardingChecklist.tsx:60]: I commented it out for now

File: frontend/src/screens/wallet/OnboardingChecklist.tsx (hunk 2 above)
Comment (rolznz) [reply frontend/src/screens/wallet/OnboardingChecklist.tsx]: fixed

**Comment by rolznz** [comment]
@reneaaron I made some improvements, please review. What do you think about the hover state? if not I can revert the change

**Comment by reneaaron** [comment]
Looks great, I added some minor improvements now, LGTM!

**Pull request merged**